docs/content/error/orderBy/notarray.ngdoc

```diff
@@ -0,0 +1,50 @@
+@ngdoc error
+@name orderBy:notarray
+@fullName Not an array
+@description
+
+This error occurs when {@link ng.orderBy orderBy} is not used with an array:
+```html
+<input ng-model="search">
+<div ng-repeat="(key, value) in myObj | orderBy:search">
+  {{ key }} : {{ value }}
+</div>
+```
+
+orderBy must be used with an array so a subset of items can be returned.
```

Comment:
`orderBy must be used with an array-like input so a subset of items can be returned.`